specs/README.md

```diff
@@ -1,84 +1,164 @@
 # Project Reunion Specs
 
-This directory contains archived and in-progress spec documents for APIs in
+This directory contains archived and in-progress spec documents for APIs in Project Reunion.
+
+Documents in this directory are used as part of a feature or API review for features in development
+and form the basis for public documentation and sample code. Write specs so they can be directly
+consumed by an app developer or document author.
+
+**DO** Provide feedback by **commenting on active pull requests** tagged with
+[the "api-design" label.](https://github.com/microsoft/ProjectReunion/pulls?q=is%3Apr+is%3Aopen+label%3Aapi-design)
+
+**DO** Provide feedback by **opening
+[issues](https://github.com/microsoft/ProjectReunion/issues/new/choose) regarding spec documents in
+this folder**. Specs in this folder are ready for implementation or have already been added to
 Project Reunion.
 
-Documents in this directory are used as part of a feature or API review
-for features in development and form the basis for public documentation
-and sample code. Write specs so they can be directly consumed by an
-app developer or document author.
+**DO** Create pull requests for new or updated specs when requested to do so as part of an issue
```

Comment:
So I guess that in a feature iteration, we could presumably modify an older spec and asend a new PR? say we need to add a new API to an existing feature set. Or does that go in as a new feature once we create a new issue?

Reply:
The specs in this directory should be "live" - if they get new features, update the existing pull requests rather than making entirely new ones.